src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html

```diff
@@ -108,13 +137,29 @@ <h2>Request for judgment by default</h2>
   <div class="section-body">
     <table>
       <tr>
-        <td>Full Name:</td>
+        <td><strong>Name:</strong></td>
```

Comment:
I used the same approach as in latest sealed copy of claim. Prototype introduces some new approach with `Company name`, `Organisation name` and `Contact name`. It does not give example for sole traders and individuals.

I will clarify that tomorrow and either keep current format or implement new format when I have more details how to do so.